<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtPortTerm</class>
 <widget class="QMainWindow" name="mdtPortTerm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MDT Port Terminal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbTerm">
        <property name="text">
         <string>RX data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClearTerm">
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTextEdit" name="teTerm">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbCmd">
      <property name="text">
       <string>Command:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="leCmd"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pbSendCmdAbort">
      <property name="text">
       <string>Abort</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pbSendCmd">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Port">
    <property name="title">
     <string>&amp;Port</string>
    </property>
    <addaction name="action_Setup"/>
    <addaction name="separator"/>
    <addaction name="action_SerialPort"/>
    <addaction name="action_UsbTmcPort"/>
   </widget>
   <widget class="QMenu" name="menu_Language">
    <property name="title">
     <string>&amp;Language</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Port"/>
   <addaction name="menu_Language"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Setup">
   <property name="text">
    <string>&amp;Setup</string>
   </property>
  </action>
  <action name="action_French">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;French</string>
   </property>
  </action>
  <action name="action_German">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;German</string>
   </property>
  </action>
  <action name="action_SerialPort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Serial port</string>
   </property>
  </action>
  <action name="action_UsbTmcPort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>USBTMC port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>activated()</signal>
   <receiver>mdtPortTerm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
